python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -311,7 +327,8 @@ def _calc_stats(input):
                                  2, 1)
 
         # Read system timestamps from event notifications, if present.
-        result = self.reader.read(message_types=[EventNotificationMessage], **self.params)
+        result = self.reader.read(message_types=[EventNotificationMessage], source_ids=self.source_ids,
```

Comment:
Here and below, the majority of these message types don't care about source IDs and should probably just read and display all. This one for example. You should be able to simply not set `source_ids` at all for these. The default behavior should be to read all available sources.

The only read calls that need to set the source ID list I think should be pose and pose aux currently.